app/views/devise/sessions/_login_form.html.haml

```diff
@@ -1,21 +1,20 @@
-.login-card
-    .card-background
-        .card-text
-            %h2 Log in
-        = form_with(model: resource, as: resource_name, url: session_path(resource_name), id: :user_login) do |f|
-            #error_messages
+.card-background
+    .card-text
+        %h2.text-grey-darkest Log in
+    = form_with(model: resource, as: resource_name, url: session_path(resource_name), id: :user_login) do |f|
+        #error_messages
+        .field
+            = f.label :email, class: "email-label"
+            %br/
+            = f.email_field  :email, class: "email-field", autofocus: true, autocomplete: "email"
+        .field
+            = f.label :password, class: "email-label"
```

Comment:
Why did you call the class email-label if it’s used on all labels?

Reply:
Was there so I didn't change it. Clearly wrong

Reply:
fixed